swatch-producer-azure/src/main/resources/application.properties

```diff
@@ -99,6 +100,10 @@ mp.messaging.incoming.tally-in.auto.offset.reset = earliest
 mp.messaging.outgoing.tally-out.connector=smallrye-kafka
 mp.messaging.outgoing.tally-out.topic=platform.rhsm-subscriptions.billable-usage
 
+# Dead Letter Topic settings
+mp.messaging.outgoing.tally-dlt.connector=smallrye-kafka
```

Comment:
I would rename this to `billable-usage-dlt` instead of `tally-dlt` since its going to be billable-usage messages.

Reply:
I named it "tally-dlt" to be in sync with the `tally-in` and `tally-out` channels. But I can change these others too, wdyt?

Reply:
oh good point, I think it makes more sense as billable-usage but i'm okay with leaving it as is in this case.